pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -2401,15 +2401,12 @@ func TestNodeShouldRunDaemonPod(t *testing.T) {
 
 // DaemonSets should be resynced when node labels or taints changed
 func TestUpdateNode(t *testing.T) {
-	var enqueued bool
 	cases := []struct {
-		test               string
-		newNode            *v1.Node
-		oldNode            *v1.Node
-		ds                 *apps.DaemonSet
-		expectedEventsFunc func(strategyType apps.DaemonSetUpdateStrategyType) int
-		shouldEnqueue      bool
-		expectedCreates    func() int
```

Comment:
While `expectedEventsFunc` seem to always return 0 (so it's fine to remove it), why do you remove it?

Reply:
Mostly - yes, but there's at least [one case](https://github.com/kubernetes/kubernetes/blob/bce5886c975c933c02583213f72ea9a27c9e2296/pkg/controller/daemon/daemon_controller_test.go#L2450-L2473) where we modify node's resources (from 100M,100m to 200M,200m) which should allow the DS to create the pod (which requires 200M,200m). You can change that being a function to just being an int, but I'd prefer we keep that case tested correctly.

Reply:
It's `expectedCreates` that is 1 there - which is what I'm asking about.
[I was saying that `expectedEventsFunc` (the function from 2 lines above is always 0 and removing that one seems fine].

Reply:
Sorry, you're right, I misread that :wink:

Reply:
Done.